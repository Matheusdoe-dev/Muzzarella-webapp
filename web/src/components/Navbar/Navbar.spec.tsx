import { render, screen } from "@testing-library/react";

import { Navbar } from "..";

describe("Navbar component", () => {
  it("should render", () => {
    render(<Navbar />);

    expect(screen.getByTestId("navbar")).toBeInTheDocument();
  });

  it("should have four navigation options", () => {
    render(<Navbar />);

    expect(screen.getByTestId("navbar")).toMatchSnapshot();
    expect(screen.getAllByTestId("navbar-option").length).toBe(4);
  });
});
