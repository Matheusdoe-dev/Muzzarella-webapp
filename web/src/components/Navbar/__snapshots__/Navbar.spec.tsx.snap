// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component should have four navigation options 1`] = `
<nav
  data-testid="navbar"
>
  <ul
    class="navbar-list"
  >
    <li
      data-testid="navbar-option"
    >
      <a
        class="navbar-list__option false"
        href="/"
      >
        Home
      </a>
    </li>
    <li
      data-testid="navbar-option"
    >
      <a
        class="navbar-list__option false"
        href="/menu"
      >
        Menu
      </a>
    </li>
    <li
      data-testid="navbar-option"
    >
      <a
        class="navbar-list__option false"
        href="/about"
      >
        About
      </a>
    </li>
    <li
      data-testid="navbar-option"
    >
      <a
        class="navbar-list__option false"
        href="/faq"
      >
        FAQ
      </a>
    </li>
  </ul>
</nav>
`;
